projects/common/src/utilities/validators/form-validators.ts

```diff
@@ -0,0 +1,13 @@
+import { AbstractControl } from '@angular/forms';
+import { getStringsFromCommaSeparatedList } from '../formatters/string/string-formatter';
+import { areEmailAddressesValid } from './email-validator';
+
+// tslint:disable-next-line: no-namespace
+export namespace CustomFormValidators {
+  export const commaSeparatedEmails = (control: AbstractControl) => {
```

Comment:
I think we should just call it `email` and also handle the `single` or `multiple` (comma separated) case in the same validator.

```ts
const emailControl = new FormControl('', [HtValidators.email])
```

Reply:
Not sure, won't it reduce its readability? It can validate single email too though

Reply:
Because there can be many ways emails can land up here if we call it just email(s), single email, comma separated emails, array of emails (which it is not ready for yet), etc